src/Hl7.Fhir.Base/ElementModel/TypedElementParseExtensions.cs

```diff
@@ -90,7 +90,7 @@ public static Quantity ParseQuantity(this ITypedElement instance)
 
         [Obsolete("WARNING! Intended for internal API usage exclusively, interface IBaseElementNavigator can be changed in " +
             "the near future.")]
-        public static Quantity ParseQuantityInternal<T>(this IBaseElementNavigator<T> instance) where T : IBaseElementNavigator<T>
+        public static Quantity ParseQuantityInternal(this ITypedElement instance) 
```

Comment:
The introduction of IBaseElementNavigator made it necessary to make these public, I think they can be internal again - in any case the reference to IBaseElementNavigator in the Obsolete should be changed!